.github/workflows/ci.yml

```diff
@@ -60,14 +60,9 @@ jobs:
         shell: bash
         run: |
            sudo apt-get update
-           sudo apt-get install podman bats bash codespell python3-argcomplete pipx git cmake
+           sudo apt-get install podman bats bash codespell python3-argcomplete pipx git cmake curl
```

Comment:
What we actually want is this, curl is just the CLI tool:

libcurl4-openssl-dev